packages/core/upload/server/services/provider.js

```diff
@@ -4,6 +4,12 @@ const { isFunction } = require('lodash/fp');
 const { streamToBuffer } = require('../utils/file');
 
 module.exports = ({ strapi }) => ({
+  async generateFileName(filename) {
```

Comment:
this function should call the generic generateFileName if it's not found in the provider

Reply:
> this function should call the generic generateFileName if it's not found in the provider

@alexandrebodin But if call the generic generateFileName, we need export this function in [upload.js](https://github.com/strapi/strapi/blob/main/packages/core/upload/server/services/upload.js), is it right?